packages/compilers/src/lib/vyperCompiler.ts

```diff
@@ -37,8 +35,8 @@ export function findVyperPlatform(): string | false {
 export async function useVyperCompiler(
   vyperRepoPath: string,
   version: string,
-  vyperJsonInput: VyperJsonInput,
-): Promise<VyperOutput> {
+  vyperJsonInput: any,
```

Comment:
In the new verification flow PR there are a lot of new types like this one. I'm wondering if we should move them in this new package. Also I see here you removed the type, why?

Reply:
Mhhh I already don't like this :D The only solution I see other than including the types in the compiler package is to create another `compiler-types` package

Reply:
I'm not sure how to move forward here. Another package for the types would definitely solve this, but it also seems a bit too much to me. @kuzdogan What do you think about this issue?

Reply:
We discussed this shortly in the call and the conclusion was to use any for now here but open another issue to export types to a new package. 

Correct me if I'm wrong.

Reply:
issue: #1958